### PR TITLE
Create test113

### PR DESCRIPTION
**Pull request by anhdh-3069**
ok

### CODE REVIEW

**Comment by anhdh-3069** [comment]
ready

**Comment by anhdh-3069** [comment]
ready

**Comment by anhdh-3069** [comment]
ready

**Comment by manhnt-0960** [comment]
commented

File: test113 [hunk 1]
```diff
@@ -0,0 +1 @@
+ahihi
```
Comment (manhnt-0960) [inline test113:1]: cmt

**Comment by manhnt-0960** [comment]
ready

**Comment by manhnt-0960** [comment]
ready

**Pull request closed without merging**